installing/upgrade/upgrade-to-r3.2.md

```diff
@@ -151,7 +151,7 @@ repeated in **all** the user's Template and Standalone VMs.
 
 4.  Add new packages (only needed in default template):
 
-        sudo dnf install qubes-mgmt-salt-vm-connector
+        sudo apt-get install qubes-mgmt-salt-vm-connector
```

Comment:
Actually `apt-get` should be added, not replaced, as it isn't available on Fedora template. It depends on what is default template.